### ISSUE TITLE
Length of writingPrompts test set outputs

### ISSUE THREAD

**Issue by cairoHy** [pytorch/fairseq#1227]
Hi, this repo provides test set outputs samples here: https://github.com/pytorch/fairseq/tree/master/examples/stories, but all outputs are trimmed to about 150 tokens, Is it possible to provide the complete outputs of the fusion model and Conv-Seq2Seq model, thx~

**Comment by huihuifan** [comment]
Thanks for your interest. Responding here so others can see rather than in your email. In our original work, the output was kept to 150 tokens to ease human evaluation. We found that it was difficult to conduct low-variance experiments with human evaluators when the stories were too long. If you would like to generate longer stories, you can use the pre-trained model that we've open sourced, see the examples folder. Thanks!
